specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2020-08-13-aml-preview/storagecache.json

```diff
@@ -2656,6 +2617,28 @@
       "required": true,
       "type": "string",
       "description": "Client API version."
+    },
+    "amlFileySystemArchiveParameter": {
+      "description": "Name for the amlFilesystem.  Alphanumerics, underscores, and hyphens. Start and end with alphanumeric.",
+      "in": "path",
+      "name": "amlFilesystemName",
```

Comment:
This looks cut-and-paste

Reply:
Its similar to the field description but its a parameter which has "in" : "path" which won't work in a definition.  Also x-ms-parameter-location